js/src/apbct-public--2--public.js

```diff
@@ -1648,6 +1648,28 @@ function apbctRealUserBadgeViewPopup(id) {
     }
 }
 
+/**
+ * Closes The Real Person popup when the cursor leaves the badge element area
+ * @param {object} event
+ */
+// eslint-disable-next-line no-unused-vars,require-jsdoc
+function apbctRealUserBadgeClosePopup(event) {
```

Comment:
Please, add the `@todo` mark to replace this logic into the css in the future refactoring.

Reply:
ok